docs/compute-engine/build_android.md

```diff
@@ -0,0 +1,78 @@
+## Create an LCE inference binary for Android ##
+!!! note
+    This guide explains how to build standalone inference binary that can be executed on Android OS. If you'd rather use LCE for inference as part of your own Android app, please check out [this guide](/compute-engine/quickstart_android) instead.
+
+To build Larq Compute Engine (LCE) for Android,
+you must have the [Android NDK](https://developer.android.com/ndk) and
+[SDK](https://developer.android.com/studio) installed on your system.
+Below we explain how to install the Android prerequisites in the LCE
+Docker container and how to configure the LCE Bazel build settings
+accordingly. Before proceeding with the next steps, please follow
+the instructions in the main [LCE build guide](/compute-engine/build/) to setup
+the Docker container for LCE and the Bazel build system.
+
+NOTE: we recommend using the docker volume as described in the
+[LCE build guide](/compute-engine/build/) to be able to easily transfer
+files in-between the container and the host machine.
+
+##### Install prerequisites #####
+
+We provide a bash script which uses the `sdkmanager` tool
+to install the Android NDK and SDK inside the Docker container.
+Please run the script by executing the following command from the LCE
+root directory:
+
+```bash
+./third_party/install_android.sh
+```
+
+After executing the bash script, please accept the Android SDK licence agreement.
+The script will download and unpack the Android NDK and SKD under the directory
+`/tmp/lce_android` in the LCE docker container.
+
+##### Custom android version #####
+
+The Android NDK and SDK versions used in LCE are currently hard-coded in the
+install script.
+To build LCE against a different NDK and SDK versions, you can manually
+modify `ANDROID_VERSION` and `ANDROID_NDK_VERSION` variables in the
+install script. Additionally, the following configurations in `configure.sh`
+(or `.bazelrc`) file need to be adjusted:
+
+```shell
+build --action_env ANDROID_NDK_HOME="/tmp/lce_android/ndk/19.2.5345600"
+build --action_env ANDROID_NDK_API_LEVEL="21"
+build --action_env ANDROID_BUILD_TOOLS_VERSION="28.0.3"
+build --action_env ANDROID_SDK_API_LEVEL="23"
+build --action_env ANDROID_SDK_HOME="/usr/local/android/android-sdk-linux"
```

Comment:
These values have been updated since the guide was originally written, and now live just in the `.bazelrc`.

```suggestion
install script. Additionally, the following configurations in the `.bazelrc`
file need to be adjusted:

```shell
build --action_env ANDROID_NDK_HOME="/tmp/lce_android/ndk/18.1.5063045"
build --action_env ANDROID_NDK_API_LEVEL="21"
build --action_env ANDROID_BUILD_TOOLS_VERSION="28.0.3"
build --action_env ANDROID_SDK_API_LEVEL="29"
build --action_env ANDROID_SDK_HOME="/tmp/lce_android"
```